app/javascript/data/models/plans.js

```diff
@@ -45,3 +45,14 @@ export const plansSchema = array()
     })
   )
   .nullable();
+
+export const PLAN_JOB_STATES = {
```

Comment:
I believe both`PLAN_JOB_STATES` and `PLAN_JOB_STATUS` ought to live in `PlanConstants.js`  

These files 🏠  all the schema bits 🤔 not so much any constants

Reply:
But maybe we oughta have an app wide constants file... that covers both plan and overview components (keepin it dry 💦 ) cuz... these things look to be pretty universal

Reply:
throughout the app we have 11 uses (not counting this pr) of things that match `status ===` 🤷‍♂️ 

ALSO maybe that's not work to do in this pr, consolidate these constants... but for sure they should be moved outta schema 📁